src/index.cts

```diff
@@ -127,11 +73,15 @@ export type Context = {
   identityClaim: string[]
 }
 
-export type EncryptedEqlPayload = {
+export type Encrypted = {
```

Comment:
Should we lift this login in Protect.js to this repo? https://github.com/cipherstash/protectjs/blob/main/packages/protect/generateEqlSchema.ts

Then leverage the EqlSchema upstream?

Reply:
Nice. Yes, I like that approach. Let's add that in in a follow-up PR.